system-tests/project-fixtures/react/cypress-vite.config.ts

```diff
@@ -9,12 +10,25 @@ declare global {
   }
 }
 
+const port = 8888
+
+const viteConfig = require('./webpack.config.js')
+
+viteConfig.server ??= {}
```

Comment:
does this have the same effect as `viteConfig.server = viteConfig.server ?? {}`?

Reply:
Yes, https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_assignment